src/instrumenter/instrumentation_context.ts

```diff
@@ -243,5 +282,166 @@ export class InstrumentationContext {
         for (const transCtx of this.transCtxMap.values()) {
             transCtx.finalize();
         }
+
+        for (const unit of this.unitsNeedingUtils) {
+            const path = relative(dirname(unit.absolutePath), this.utilsUnit.absolutePath);
+            unit.appendChild(
+                this.factory.makeImportDirective(
+                    `./${path}`,
+                    this.utilsUnit.absolutePath,
+                    "",
+                    [],
+                    unit.id,
+                    this.utilsUnit.id
+                )
+            );
+        }
+    }
+
+    private customMapLibrary = new Map<string, CustomMapLibraryMD>();
```

Comment:
Can this be moved to a class header variable sequence?

Reply:
Fixed in a97c7d9.